### PR TITLE
#21089 checkbot plugins tooltips

### PR DESCRIPTION
**Pull request by vadura-pavel**

### CODE REVIEW

File: src/CheckbotPlugins/IdeaStatiCa.CheckbotPlugin.PluginList/Json/ActionButton.cs [hunk 1]
```diff
@@ -15,5 +15,8 @@ internal class ActionButton
 
 		[JsonPropertyName("tooltip")]
 		public string Tooltip { get; set; } = string.Empty;
+
+		[JsonPropertyName("tooltip_dissable")]
```
Comment (dbacovsky) [inline src/CheckbotPlugins/IdeaStatiCa.CheckbotPlugin.PluginList/Json/ActionButton.cs:19]: Melo by to byt spis tooltip_disabled

File: src/CheckbotPlugins/IdeaStatiCa.CheckbotPlugin.PluginList/Json/OpenButton.cs [hunk 2]
```diff
@@ -17,6 +16,9 @@ internal class OpenButton
 		[JsonPropertyName("tooltip")]
 		public string Tooltip { get; set; } = string.Empty;
 
+		[JsonPropertyName("tooltip_dissable")]
```
Comment (dbacovsky) [inline src/CheckbotPlugins/IdeaStatiCa.CheckbotPlugin.PluginList/Json/OpenButton.cs:19]: preklep, tooltip_disabled

Comment (dbacovsky) [inline src/CheckbotPlugins/IdeaStatiCa.CheckbotPlugin.PluginList/Json/OpenButton.cs:19]: jeste tady

Comment (vadura-pavel) [reply src/CheckbotPlugins/IdeaStatiCa.CheckbotPlugin.PluginList/Json/OpenButton.cs]: fixed

**Review by dbacovsky — APPROVED** [review]

**Pull request merged**